Added get_grade_book function

kchattx opened the pull request:
Added get_grade_book function so users can retrieve the grades of their current classes without having to parse and iterate through all of them.

kchattx committed:
Update studentvue.py

kajchang commented:
Thanks! A few things: 
1. Don't use format strings because they're not compatible with versions earlier than Python 3.7
2. Make the return documentation more descriptive
3. Change the values in the return dictionary from lists to a dictionaries that label the mark and score

kchattx committed:
Update get_grade_book function

1. Removed format strings and replaced it with .format()
2. Made the return documentation more descriptive (I think...)
3. Changed the lists in the return values containing mark and score to dictionaries

IronFlame07 commented:
When you say "make the return documentation more descriptive", what exactly does that mean? More description of the structure of the return value, or more descriptive about what information it holds?
